Core/include/Acts/EventData/TrackParametersConcept.hpp

```diff
@@ -26,24 +26,32 @@ using TypeParametersVector = typename T::ParametersVector;
 template <typename T>
 using TypeCovarianceMatrix = typename T::CovarianceMatrix;
 
-template <typename T>
-using ReturnTypeReferenceSurface =
-    decltype(std::declval<T>().referenceSurface());
 template <typename T>
 using ReturnTypeParameters = decltype(std::declval<T>().parameters());
 template <typename T>
 using ReturnTypeCovariance = decltype(std::declval<T>().covariance());
 template <typename T>
+using ReturnTypePosition4FromContext =
+    decltype(std::declval<T>().position4(GeometryContext()));
```

Comment:
This should use `std::declval`. `GeometryContext` is a typedef over `std::any`, and while that is actually default-constructible, we should not require it. Also, there's no implicit meaning that *empty* `GeometryContext` means *nominal alignment*.

Reply:
Fixed.